app.js

```diff
@@ -0,0 +1,23 @@
+const express = require("express");
+const {
+  catchAll,
+  handlePSQLErrors,
+  handleCustomErrors,
+} = require("./controllers/errors.controller");
+const { getCategories } = require("./controllers/games.controller");
+
+const app = express();
+
+app.use(express.json());
+

```

Comment:
We don't need 22 here quite yet (only when we start posting), remove it for now, only add it when needed.